<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelVoltage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>                               TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="tensionSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="transmittedTension">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tensionSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>setTension(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>onStopButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setTension(int)</slot>
  <slot>onStopButtonClicked()</slot>
 </slots>
</ui>
